app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -26,6 +26,26 @@ def textual_name
     {:label => "Name", :value => @record.name}
   end
 
+  def textual_phase
+    {:label => "Phase", :value => @record.phase}
+  end
+
+  def textual_message
+    if @record.message.nil?
+      nil
+    else
+      {:label => "Message", :value => @record.message}
+    end
```

Comment:
I think it can be shrunk to:

    def textual_message
        {:label => "Message", :value => @record.message} unless @record.message.nil?
    end